packages/react/src/VideoPlayer/VideoPlayer.module.css

```diff
@@ -0,0 +1,399 @@
+  padding: 1px 5px;
+  border: 2px solid var(--base-color-scale-gray-0);
+  background: var(--base-color-scale-gray-0);
+  position: relative;
+}
+
+.VideoPlayer__ccOff {
+  background-color: transparent;
+}
+
+.VideoPlayer__ccText {
+  color: var(--base-color-scale-black-0);
+}
+
+.VideoPlayer__ccOff .VideoPlayer__ccText {
+  color: var(--base-color-scale-gray-0);
+}
+
+/* ---------------------------------------------------------- */
+/* 6. Range Element                                           */
+/* ---------------------------------------------------------- */
+.VideoPlayer__range {
+  position: relative;
+  width: 100%;
+  min-width: var(--brand-VideoPlayer-range-minWidth);
+  height: var(--brand-VideoPlayer-range-height);
+  background-color: var(--base-color-scale-gray-0);
+  border-radius: var(--brand-VideoPlayer-range-borderRadius);
+}
+
+.VideoPlayer__container--small .VideoPlayer__volumeBar {
+  display: none;
+}
+
+.VideoPlayer__range:hover .VideoPlayer__tooltip {
+  opacity: 1;
+}
+
+.VideoPlayer__rangeInput {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  z-index: 1;
+  display: block;
+  width: 100%;
+  height: var(--brand-VideoPlayer-range-height);
+  margin: 0;
+  cursor: pointer;
+  background: transparent;
+  position: absolute;
+  top: 0;
+  opacity: 0;
+}
+
+.VideoPlayer__rangeInput:focus-visible {
+  border-color: var(--brand-color-focus);
+  outline: none;
+  box-shadow: 0 0 0 2px var(--brand-color-focus);
+  opacity: 1;
+}
+
+.VideoPlayer__range:hover .VideoPlayer__rangeInput {
+  opacity: 1;
+}
+
+.VideoPlayer__rangeInput::-moz-range-track,
+.VideoPlayer__rangeInput::-webkit-slider-runnable-track {
+  background: transparent;
+  border: none;
+}
+
+.VideoPlayer__rangeProgress {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  overflow: hidden;
+  background: hsl(var(--base-color-scale-gray-0-hsl) / 50%);
+  border: 0;
+  border-radius: var(--brand-VideoPlayer-range-borderRadius);
+  width: 100%;
+  position: absolute;
+  height: var(--brand-VideoPlayer-range-height);
+  top: 0;
+}
+
+.VideoPlayer__rangeProgress::-webkit-progress-bar {
+  background-color: hsl(var(--base-color-scale-gray-0-hsl) / 50%);
+}
+
+.VideoPlayer__rangeProgress::-webkit-progress-value {
+  background-color: var(--base-color-scale-blue-5);
+}
+
+.VideoPlayer__rangeProgress::-moz-progress-bar {
+  background: var(--base-color-scale-blue-5);
+}
+
+.VideoPlayer__rangeProgress::-ms-fill-lower {
+  background-color: var(--base-color-scale-blue-5);
+  border: none;
+}
+
+.VideoPlayer__rangeProgress::-ms-fill-upper {
+  background-color: hsl(var(--base-color-scale-gray-0-hsl) / 50%);
+  border: none;
+}
+
+/* ---------------------------------------------------------- */
+/* 7. Tooltip                                                 */
+/* ---------------------------------------------------------- */
+
+.VideoPlayer__tooltip {
+  position: absolute;
+  bottom: 100%;
+  /* left: 50%; */
```

Comment:
can you remove this if it's not needed?

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/373/commits/53e6be6f783cb211109e7502de6eb28f3b6699e4